### DISCUSSION TITLE
Why doesn't the Security/Login API work

### DISCUSSION THREAD

**Discussion by luo123qiu** [apache/superset#13663] [Q&A / Help]
I tried to use the Security/login API, but it showed 404 when I accessed it, while other APIs were still working. Can someone help me?Thank you

**Comment by yangtian9999** [comment]
I also encounted this issue today, still can not find the solution

**Comment by pritypriya25** [comment]
Are you passing the following request body?
{
  "password": "complex-password",
  "provider": "db",
  "refresh": true,
  "username": "admin"
}
The username and password must match with the one that you enter for dashboard login

**Reply by yangtian9999** [reply]
I made mistake for the address. the correct address is     http://xxxx/api/v1/security/login
and must set FAB_API_SWAGGER_UI: "true"  when first time init the superset

**Reply by pritypriya25** [reply]
ohk .. that can also be a reason for error. 
I am running apache/superset on my docker container. I want to hit the public rest api of superset. The api/v1/{pk}/cache_screenshot  will provide me the image url but even after hitting the api I am getting error 404 : Not found .. Can you help me with that ?
